spec/requests/api/v1/settings_spec.rb

```diff
@@ -1,39 +1,46 @@
+# frozen_string_literal: true
+
 require 'rails_helper'
 
-describe 'GET api/v1/settings/must_update', type: :request do
+describe 'GET api/v1/settings/must_update' do
+  subject(:endpoint) { get must_update_api_v1_settings_path, params: }
+
   let(:params) do
     {
       device_version: '1.0'
     }
   end
-  subject { get must_update_api_v1_settings_path, params: }
 
   context 'with correct settings' do
-    let!(:setting_version) { create(:setting_version) }
+    before do
+      create(:setting_version)
+    end
 
     it 'return success' do
-      subject
+      endpoint
       expect(response).to be_successful
     end
 
     it 'returns no need to update' do
-      subject
+      endpoint
       expect(json['must_update']).to be(false)
     end
   end
 
   context 'with incorrect settings' do
-    let!(:setting_version) { create(:setting_version, value: '2.0') }
+    before do
+      create(:setting_version, value: '2.0')
+    end
```

Comment:
Why this change? Just asking

Reply:
I don't remember the name of the cop now, but there is one where It detects an infraction when you declare `let!` and you don't use it how happens in this example, so you should use `before`